source/hello/src/hello.c

```diff
@@ -0,0 +1,7 @@
+#include <stdio.h>
```

Comment:
That's absolutely fine but we will not need these files in the forked projects so we may get rid of this file and we might print "Hello World" message in the main.c. Otherwise, all forked project will need to delete this file and its header.

Reply:
Probably we will have a framework for all projects. This added as to check folder structure. With this hello Cmake should move to add_directory command